From 367f3378cc579e8dd131ffcc717a337915e06519 Mon Sep 17 00:00:00 2001
From: Alban Browaeys <prahal@yahoo.com>
Date: Tue, 10 Apr 2012 10:35:18 +0000
Subject: install : fix install with a DESTDIR (jhbuild)

Spurious double quotes around uidir and iconsdir makes
install attempt to install in '<DESTDIR>"<iconsdir>"'
ie
'/opt/gnome/_jhbuild/root-libgnomekbd\"/opt/gnome/share/libgnomekb/icons\"'

Then install fails to install the data files into the correct directory:
'/opt/gnome/_jhbuild/root-libgnomekbd/opt/gnome/share/libgnomekb/icons'.
It looks like the for loop to install items expands the icons and ui
dirs while the mkdir -p does not.

Fix this issue by removing the double quotes in th Makefile.am around
the iconsdir and uidir assignments.
---
diff --git a/libgnomekbd/Makefile.am b/libgnomekbd/Makefile.am
index ed71260..6246760 100644
--- a/libgnomekbd/Makefile.am
+++ b/libgnomekbd/Makefile.am
@@ -2,10 +2,10 @@ bin_PROGRAMS=gkbd-keyboard-display
 
 lib_LTLIBRARIES = libgnomekbd.la libgnomekbdui.la
 
-uidir="$(pkgdatadir)/ui"
+uidir=$(pkgdatadir)/ui
 ui_DATA = show-layout.ui
 
-iconsdir="$(pkgdatadir)/icons"
+iconsdir=$(pkgdatadir)/icons
 icons_DATA = kbdled-caps-lock.svg \
              kbdled-num-lock.svg \
              kbdled-scroll-lock.svg 
--
cgit v0.9.0.2
